backend/package.json

```diff
@@ -11,7 +11,8 @@
     "cors": "^2.8.5",
     "debug": "~2.6.9",
     "express": "~4.16.0",
-    "firebase-admin": "^6.4.0",
+    "firebase": "^5.8.0",
```

Comment:
wyrzucić daszki aby zostalo 5.8.0 i 6.5.0 
Jesli tak zrobienie to zawsze bedziecie mieli ta wersje po npm install. Jesli zostawiecie to npm install zupdatuje to i moze wam przestac działac wszystko z FB